### ISSUE TITLE
Feature request HHVM

### ISSUE THREAD

**Issue by XoneFobic** [puphpet/puphpet#399]
_TL;DR at the bottom_

Hey guys,

I've got another feature request.

I use Laravel as my framework. And they use Travis-CI for testing.
Now I read through the travis file in their repo and came across HHVM.
So I've been reading up on it.

And I have to say it looks pretty interesting.
It's a PHP replacement build in C by the people over at facebook. And should speed up load time a lot.

I got it running now on a modified PuPHPet box.
But I have no idea how you build your script. So I can't build it in and then to a pull request.

**TL;DR**

So, my feature request is. To be able to select HHVM as a PHP version when using NGINX.
The link I used to get mine up and running is http://fideloper.com/hhvm-nginx-laravel.
Let me know what you think.
Install was pretty straigth forward and I would have implemented it, if I knew how you build PuPHPet. (Never touched Ruby before...)
